spec/support/shared/unit/script_resource.rb

```diff
@@ -47,9 +47,9 @@
   end
 
   describe "when executing guards" do
-    it "inherits exactly the :cwd, :environment, :group, :path, :user, and :umask attributes from a parent resource class" do
+    it "inherits exactly the :cwd, :domain, :environment, :group, :password, :path, :user, and :umask attributes from a parent resource class" do
       inherited_difference = Chef::Resource::Script.guard_inherited_attributes -
-        %i{cwd environment group path user umask}
+        %i{cwd domain environment group password path user umask}
 
       expect(inherited_difference).to eq([])
     end
```

Comment:
Super nitpicky, but both of these tests should probably be something like:

```
expected = %i{cwd domain environment group password path user umask}
expect(Chef::Resource::Script.guard_inherited_attributes).to eq(expected)
```

Reply:
Unless there's some subtlety I'm missing

Reply:
wouldn't that fail if `expected` got the ordering wrong?

Reply:
Good point, it should be `match_array` in that case.